index.js

```diff
@@ -74,6 +74,12 @@ module.exports = function (options, cb) {
       return taskRunner.runPromisified(params);
     })
     .then((taskDefinition) => {
+      if (taskDefinition.failures) {
+        console.error("Task failed to launch:")
```

Comment:
I think it's fine to just throw the error - anything interested should catch it and log as appropriate. This is intended to be used as a library after all.

Reply:
Will do